src/routers/datatypeRouters/edges/variants_regulatory_regions.ts

```diff
@@ -4,7 +4,7 @@ import { QUERY_LIMIT, configType } from '../../../constants'
 import { publicProcedure } from '../../../trpc'
 import { loadSchemaConfig } from '../../genericRouters/genericRouters'
 import { distanceGeneVariant, getFilterStatements, paramsFormatType, preProcessRegionParam } from '../_helpers'
-import { HS_ZKD_INDEX, MM_ZKD_INDEX } from '../nodes/regulatory_regions'
+import { HS_ZKD_INDEX, MM_ZKD_INDEX } from '../nodes/genomic_elements'
```

Comment:
This file variants_regulatory_regions.ts doesn't seems to be updated.

Reply:
good catch... just pushed the changes